src/routes/(console)/organization-[organization]/settings/deleteOrganizationEstimation.svelte

```diff
@@ -0,0 +1,31 @@
+<script lang="ts">
+    import { Alert } from '$lib/components';
+    import type { EstimationDeleteOrganization } from '$lib/sdk/billing';
+    import RetryPaymentModal from '../billing/retryPaymentModal.svelte';
+    import { selectedInvoice, showRetryModal } from '../billing/store';
+    import InvoicesTable from './invoicesTable.svelte';
+
+    let error: string = '';
+    export let estimation: EstimationDeleteOrganization;
+</script>
+
+{#if error}
+    <Alert type="error">
+        <span>{error}</span>
+    </Alert>
+{/if}
+{#if estimation}
+    {#if estimation.unpaidInvoices?.length > 0}
+        <Alert type="warning">
+            <span slot="title">
+                You have a unpaid invoices. Please settle them first in order to delete team.
+            </span>
+        </Alert>
+
+        <InvoicesTable invoices={estimation.unpaidInvoices} />
+    {/if}
+{/if}
```

Comment:
I suspect this implementation will lead to bad layout shifts, let's double check

Reply:
This is still pending. As modal over modal doesn't work.